src/makeCalculator.js

```diff
@@ -37,7 +37,39 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+
+    divide(a, b) {
+      return a / b;
+    },
```

Comment:
let's handle this case here 
![image](https://github.com/mate-academy/js_advanced_calculator/assets/96138503/62b8d871-fecb-48dc-b78e-718ec9e3cd1a)